### DISCUSSION TITLE
Request to verify creators already verified on VerifiedCreators on Bitclout.com

### DISCUSSION THREAD

**Discussion by Dracklyn** [deso-protocol/dips#152] [Medium CIPs]
Hi, 

[VerifiedCreators](https://www.verifiedcreators.io/) is a native and public non-KYC verification platform for Bitclout users. Our platform combines live verification (selfie), image matching (profile photo upload), and cross-examining this information with the user's Bitclout account and other social profiles they submitted to ensure they are 100% accurate. All of the information we collect is 100% available to the general public.

Since we launched, we have been verifying Bitclout creators' identity, including some of the creators that have been here from the early days like @ElrickErikose, @MechellLord, BrixBoston100, @GJART, @DOZ, @Mashelenn, @KeleighMcCall, @transhumanist, @JennRoyster.

Our verified creators are some of the most active, committed, and supportive creators on Bitclout. Their identities have been proven legit for anyone on the internet to see or verify.

We are requesting for our verified creators to be verified globally on Bitclout.com like their counterparts. To facilitate this endeavor, we can provide direct API access to the Bitclout core team and collaborate to make this activity a seamless one.

Bitclout has given all of us the social media platform of our dreams. As part of the community, it is our responsibility to ensure Bitclout achieves its great potentials. We believe this activity is a step in the right direction to make Bitclout safer for everyone.

Thanks for your consideration.

**Comment by nigelhooijmans** [comment]
I too believe that this step is in the right direction. The verification process, besides the 100k followers on TikTok, IG or Twitter, is totally random now and decided by node admins. 

I've seen creators with <100 followers and activity <30 days with verification checks. 

I'm here for 137 days now and posting and replying each and every day - but still no verification on this side. 

BitClout, this is a step in the right direction. It's yours to take it. ✌️

**Reply by jennroyster** [reply]
Active users that have contributed daily, and have shown their value should be verified and appreciated.  They are supporting what you are creating. They believe in bitclout! I am one of them that has continuously shown up. The community is social and this is powerful for growth.

**Reply by cvos** [reply]
I hate asking this here and it’s is a tad off topic, but I’m not able to login with my seed phrase and can’t find a way to get access with my email. 

Bitclout user @cvos 

I have no idea how I locked myself out of my account

**Reply by Dracklyn** [reply]
Hi,

Sorry to hear that.

Join the community server and ask this question there. https://discord.gg/b8PA8wDr

**Comment by bradleebartlett** [comment]
Helping Bitclout creators and users who may not be solely focused on developing NFTs - or buying and selling in large quantities - is going to be the fuel that brings other users to the space. This is a great move to help increase engagement by serious community-driven users.

**Comment by theparkerazzi** [comment]
Awesome sauce.

**Reply by monurohila** [reply]
Thank you for supporting creators to get verification status on bitclout. Bitclout is fully decentralized platform still relying on verification based on these centralized platforms like FB, Twitter and Instagram.

Hope to get verified status on bitclout soon.


https://bitclout.com/u/imonurohila

**Comment by TodComplex** [comment]
Very important work you guys are doing. Keep it up!

**Comment by Butteryfly2901** [comment]
Thank you for doing all this important work. Without you I could not exist. I’m very grateful to have gotten this opportunity. Especially not being tech savvy.
I hope I’m adding joy and value to the Community.
BitcloutMusic60s70s80s
HealthyLivingRevolution

would love to get verified status.

Donna K .

**Comment by africuz** [comment]
I signed up into github just to support this initiative. 
i have been verified by VerifiedCreators, been active on Bitclout for the last 45 days.  As an active creator i think it is fair to verify us on the main Bitclout node. You guys got my full support on this.!!!

**Comment by theparkerazzi** [comment]
I support this.

On Thu, Sep 16, 2021 at 2:27 PM africuz ***@***.***> wrote:

> I signed up into github just to support this initiative.
> i have been verified by VerifiedCreators, been active on Bitclout for the
> last 45 days. As an active creator i think it is fair to verify us on the
> main Bitclout node. You guys got my full support on this.!!!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/bitclout/cips/discussions/152#discussioncomment-1343853>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEECVBHMSUUMBIMELDABRDUCIZIXANCNFSM5EE3USKQ>
> .
>
-- 
Brandon P. Parker

757-751-7791 va
917-475-6150 ny

www.theParkerazzi.com
www.instagram.com/theparkerazzi

**Comment by zachyliberos** [comment]
I support this initiative. I have a question about the payment for verified creators -there is an NFT purchase required. Are the funds used for operating costs of this initiative?

**Reply by Dracklyn** [reply]
Exactly. 50% from the NFT purchase goes to creators as creator coin royalty. The other 50% is used for the project development.

**Comment by tijno** [comment]
personally i think verification should be dropped altogether and replaced with an onchain associations / trust mechanism where you vouch for the people you know.

like google page rank.

the current verification is broken (i am verified), and people are too focussed on getting verified.

so im not in support of adding more verification providers without this onchain mechanism.

**Reply by africuz** [reply]
I think they can both be implemented simultaneously. First get verified through your socials and KYC system then after that vouching system, maybe there should be upvoting system with certain amount of votes like 3 or 5.

**Reply by Dracklyn** [reply]
Fair enough. With more people still being verified lately, I don't think Bitclout will drop verification altogether. 

A platform like Bitclout needs a transparent verification platform. A verification platform where the data is 100% available to the general public so anyone can do their due diligence.

People can game KYC platforms easily. And nobody has access to the data apart from the platform owners. For instance, in Binance, there are a bunch of verified scammers on P2P even though they have one of the best verification platforms out there.

You can say the same thing with Google page rank.  I have been scammed because of the flaws in the platform.

The idea of an on-chain association/trust mechanism where you vouch for the people you know is good. But as mentioned by @africuz,  it needs to be combined with a platform that makes it super easy for anyone to verify their identity regardless if they know Diamondhands or not. There's a significant issue with the UX of that as well. With regards to integrating it into the platform and making it easy for anyone to use.

We are only here to support the growth of creators, hence Bitclout. If there's any significant change in the current verification process, It will be good to know and innovate in that direction.


But if we are going with what we have now, then every verified creator on [VerifiedCreators ](https://www.verifiedcreators.io/)deserves to be verified on Bitclout.

**Reply by tijno** [reply]
Isnt bitclout pseudonymous ? Why should my real identity be transparant ? Next well want real names and only 1 account ? 

Why should social profiles on old world platforms determine whether you can be verified here? 

And everything can be gamed.

So networks of trust between pseudonymous ids i think is still the longer term solution.

**Reply by Dracklyn** [reply]
Pseudonymity has its good part, but it's just one aspect of Bitclout. Apart from that, it has a track record of being associated with trolls, scammers, and all sorts of negativity.

On bitclout, most of the scammers and rug pullers are pseudonymous users. 

As long as social media is concerned, the average user wants to be known and seen. With Bitclout, there's even more to that. 

There's CC and speculation, which has a lot to do with identity. Pseudonymity eliminates that idea of creator coin speculation.

**Comment by sungkhum** [comment]
Yes, there should be some sort of verification process available to everyone. I think it could be as simple as everyone who did the $10 verification process.

**Reply by arhebbar** [reply]
No offence to the effort. I think it is very well-designed. 

But, I think the minute you charge 50$ for a verification, you are biasing this significantly against the developing world. In India and I am guessing most of Asia, 50$ is a huge deal and I'd bet that this will result in a very small portion of users from Asia verified. Also why have a duplicate process for the verification. I am sure Jumio could be paid an extra 5-10$ to do the same verification since we are already sharing some of the details with them at the time of verification.

I feel like there are two options for verification:
1. Use Jumio kind of verification (ask them to be more thorough in the check - at this time, I don't think they are actually verifying, just making sure that the id is a valid id).
2. Remove verification all together and use a trust-based mechanism that @tijno recommends.

**Reply by Dracklyn** [reply]
Thank you for your feedback.

To build, maintain, and grow an app like this, requires a lot of money.
Currently, VerifiedCreators is self-funded. We charge a 0.50 verification fee. It will help us maintain and grow not just the app but the project and community as well. Not to forget, we are putting 50% of it back into the coin. 

We also have consideration for people that can afford it. All they need to do is contact us.

If you have any idea who we can get everyone in the category you mentioned verified while meeting our expectations, I am open to learning more. Get in touch.

**Reply by arhebbar** [reply]
I would make two suggestions:
1. Integrate with 1-2 test nodes and try and include it into the Verify Flow in those nodes instead of a separate standalone website and a separate flow. Getting traffic to your site will itself cost you too much money. Get the 10$ from Jumio, add your own 10-20$ and get a blue-check.
2. Try and lower your cost to something like 20$ (maybe remove the part about investing back in your coin because if this becomes profitable, everyone starts replicating your idea and you will never be the one-stop verification solution). Ideally such an effort should be non-profit (I don't mean not charging, but cover your costs including the cost of your time).

Having said that, I would still lean towards @tijno 's idea of a trust-based verification.

**Comment by theparkerazzi** [comment]
Lol. At this point everyone is on Bitclout working for free. This is comedy.

On Sun, Sep 19, 2021 at 12:00 PM Randhir Hebbar ***@***.***>
wrote:

> No offence to the effort. I think it is very well-designed.
>
> But, I think the minute you charge 50$ for a verification, you are biasing
> this significantly against the developing world. In India and I am guessing
> most of Asia, 50$ is a huge deal and I'd bet that this will result in a
> very small portion of users from Asia verified.
>
> I feel like there are two options for verification:
>
>    1. Use Jumio kind of verification (ask them to be more thorough in the
>    check - at this time, I don't think they are actually verifying, just
>    making sure that the id is a valid id).
>    2. Remove verification all together and use a trust-based mechanism
>    that @tijno <https://github.com/tijno> recommends.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/bitclout/cips/discussions/152#discussioncomment-1354020>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUEECVHDC75T4ZB6E5KNKBLUCYCJHANCNFSM5EE3USKQ>
> .
>
-- 
Brandon P. Parker

757-751-7791 va
917-475-6150 ny

www.theParkerazzi.com
www.instagram.com/theparkerazzi
